### PR TITLE
adds todo adding to App

### PR DESCRIPTION
**Pull request by FrankieFabuloso**

### CODE REVIEW

File: to-do-list/src/components/List.js [hunk 1]
```diff
@@ -2,35 +2,12 @@ import React from 'react';
...
-    const todoIndex = newState.indexOf(todo)
-    newState.splice(todoIndex,1)
-    this.setState({
-      Tasks: newState
-    })
-  }
-
   render() {
-    const todos = this.state.Tasks
+    const {todos, handleDeleteClick} = this.props
```
Comment (nicosesma) [inline to-do-list/src/components/List.js:6]: when assigning destructured variables in this manner space can make the code easier to read
```
const { todos, handleDeleteClick } = this.props
```

**Review by nicosesma — COMMENTED** [review]
looks great

File: to-do-list/src/components/List.js (hunk 1 above)
Comment (FrankieFabuloso) [reply to-do-list/src/components/List.js:6]: thanks friend! 
Were defs looking at all of your comments and really appreciate them. Were gonna try to get everything working correctly then go back and refactor our code. :]

Comment (nicosesma) [reply to-do-list/src/components/List.js:6]: awesome sounds great

**Pull request merged**